added -y option to skip prompt in release

Hypercubed opened the pull request:
Added -y prompt tio `release` cli options
Added option to skip prompts in release command
Added example to readme
Fixes  [#1]

heff commented on README.md:
````diff
@@ -121,6 +121,16 @@ grunt chg-add
+
+```json
+"scripts": {
+  "version": "version": "chg release -y && git add -A CHANGELOG.md"
````
Got a double "version" here

heff commented:
One small fix. And not sure why node 0.8 is failing in Travis but I feel like we don't care about 0.8 anymore?

Otherwise lgtm!

heff commented:
@Hypercubed added you as a collaborator, so you have push access and can accept PRs, but still make any non-doc changes through a PR that gets reviewed by one other person (a la [this governance model](http://rfc.zeromq.org/spec:22))

Hypercubed commented:
Fixed, and added a note about the required npm version.

Thanks for adding me as a collaborator.  I like what you have done here, it fits well with how I use https://github.com/Hypercubed/todo-md (old code, could use a rewrite).

Hypercubed commented:
The travis-ci test are failing at `npm update -g npm` in the node 0.8 env.  It has something to do with incompatibility with npm v3.  We can either fix the npm version (see https://github.com/Hypercubed/chg/blob/fix/travis/.travis.yml#L2) or remove node 0.8.

heff commented:
Nice, todo-md looks handy. I removed 0.8 and the before_install, so it should be ok now after a merge. Gonna go ahead and pull this in.
